platform-sdk/swirlds-common/src/main/java/com/swirlds/common/merkle/synchronization/internal/TeacherSendingThread.java

```diff
@@ -213,6 +213,7 @@ private void run() {
             throw new MerkleSynchronizationException("exception in the teacher's receiving thread", ex);
         } finally {
             senderIsFinished.set(true);
+            System.err.println("TeacherSendingThread.run finished");
```

Comment:
Nit: please either nuke this or replace with a log.

Reply:
Oh, yeah, a leftover from debugging times. Will remove